firmware/applications/SignBuddy/FreeRTOSConfig.h

```diff
@@ -7,47 +7,47 @@
-#define configUSE_MUTEXES                  1
-#define configQUEUE_REGISTRY_SIZE          8
-#define configCHECK_FOR_STACK_OVERFLOW     2
-#define configUSE_RECURSIVE_MUTEXES        1
-#define configUSE_MALLOC_FAILED_HOOK       0
-#define configUSE_APPLICATION_TASK_TAG     0
-#define configUSE_COUNTING_SEMAPHORES      1
-#define configGENERATE_RUN_TIME_STATS      0
+#define configUSE_PREEMPTION                   1
+#define configUSE_IDLE_HOOK                    0
+#define configUSE_TICK_HOOK                    0
+#define configCPU_CLOCK_HZ                     (SystemCoreClock)
+#define configTICK_RATE_HZ                     (( TickType_t ) 1000)
+#define configMAX_PRIORITIES                   (5)
+#define configMINIMAL_STACK_SIZE               (( unsigned short ) 60)
+#define configTOTAL_HEAP_SIZE                  (( size_t ) (6500))
+#define configMAX_TASK_NAME_LEN                (5)
+#define configUSE_TRACE_FACILITY               1
+#define configUSE_16_BIT_TICKS                 0
+#define configIDLE_SHOULD_YIELD                1
+#define configUSE_MUTEXES                      1
+#define configQUEUE_REGISTRY_SIZE              8
+#define configCHECK_FOR_STACK_OVERFLOW         2
+#define configUSE_RECURSIVE_MUTEXES            1
+#define configUSE_MALLOC_FAILED_HOOK           0
+#define configUSE_APPLICATION_TASK_TAG         0
+#define configUSE_COUNTING_SEMAPHORES          1
+#define configGENERATE_RUN_TIME_STATS          0
 
 /* Co-routine definitions. */
-#define configUSE_CO_ROUTINES              0
-#define configMAX_CO_ROUTINE_PRIORITIES    (2)
+#define configUSE_CO_ROUTINES                  0
+#define configMAX_CO_ROUTINE_PRIORITIES        (2)
 
 /* Software timer definitions. */
-#define configUSE_TIMERS                   1
-#define configTIMER_TASK_PRIORITY          (2)
-#define configTIMER_QUEUE_LENGTH           5
-#define configTIMER_TASK_STACK_DEPTH       (80)
+#define configUSE_TIMERS                       1
+#define configTIMER_TASK_PRIORITY              (2)
+#define configTIMER_QUEUE_LENGTH               5
+#define configTIMER_TASK_STACK_DEPTH           (80)
 
 /* Set the following definitions to 1 to include the API function, or zero
  * to exclude the API function. */
-#define INCLUDE_vTaskPrioritySet           1
-#define INCLUDE_uxTaskPriorityGet          1
-#define INCLUDE_vTaskDelete                1
-#define INCLUDE_vTaskCleanUpResources      1
-#define INCLUDE_vTaskSuspend               1
-#define INCLUDE_vTaskDelayUntil            1
-#define INCLUDE_vTaskDelay                 1
-#define INCLUDE_uxTaskGetStackHighWaterMark 1
+#define INCLUDE_vTaskPrioritySet               1
+#define INCLUDE_uxTaskPriorityGet              1
+#define INCLUDE_vTaskDelete                    1
+#define INCLUDE_vTaskCleanUpResources          1
+#define INCLUDE_vTaskSuspend                   1
+#define INCLUDE_vTaskDelayUntil                1
+#define INCLUDE_vTaskDelay                     1
+#define INCLUDE_uxTaskGetStackHighWaterMark    1
 
```

Comment:
I don't think we want to keep the GetStackHighWaterMark, that was just for debugging